### PR TITLE
add CCR for microprofile platform release

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Fixes: #213

### CODE REVIEW

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Compatible Certification Request
+about: Request certification of a compatibile implementation of MicroProfile Platform release
+title: 'Compatibility certification request for [product/project name] for MciroProfile [plaftorm release verseion]'
```
Comment (xstefank) [inline spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:4]:
```suggestion
title: 'Compatibility certification request for [product/project name] for MicroProfile [plaftorm release version]'
```

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+---
+
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable):<br/>
+  // Add here
+- [ ] Specification Name, Version and download URL:<br/>
+   // Add here
+- [ ] (Optional) TCK Version, digital SHA-256 fingerprint and download URL:<br/>
```
Comment (xstefank) [inline spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:16]: Do you mean all 8 TCKs?

Comment (Emily-Jiang) [reply spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:16]: nope. This should be only any potential TCKs added by platform. At the moment, there is none. Maybe just remove this to reduce confusion. I think asking people to type in the url for the 8 tcks are not useful.

Comment (rdebusscher) [reply spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md]: But if you file a certification request for the platform, you need to pass the different spec TCKs.  So how does this need to be provided (evidence that you pass them)?

Comment (Emily-Jiang) [reply spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md]: @rdebusscher please see the following bulletin point. You need to list all of the tck results for the 8 specs ([here](https://openliberty.io/certifications/microprofile/4.0/TCKResults.html) is an example)

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: Compatible Certification Request
+about: Request certification of a compatibile implementation of MicroProfile Platform release
+title: 'Compatibility certification request for [product/project name] for MciroProfile [plaftorm release version]'
```
Comment (ederks85) [inline spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:4]: Fix typos: MicroProfile [platform release version]

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable):<br/>
+  // Add here
+- [ ] Specification Name, Version and download URL:<br/>
+   // Add here
+- [ ] Public URL of TCK Results Summary for the included MicroProfile component specifications (Config, Fault Tolerance, OpenAPI, Rest Client, Health, Metrics, Open Tracing, JWT):<br/>
+  // Add here
+- [ ] (Optional) Any Additional Jakarta EE Specification TCKs such as the included CDI, JAX-RS, JSON-B, JSON-P, Annotation:<br/>
+  // Add here
+- [ ] Java runtime used to run the implementation:<br/>
```
Comment (ederks85) [inline spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:20]: Is it helpful to add the "Java SE 8 or higher" requirement here for clarity?

**Review by ederks85 — CHANGES_REQUESTED** [review]

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md (hunk 4 above)
Comment (Emily-Jiang) [reply spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md]: Good idea! added.

File: spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md (hunk 3 above)
Comment (Emily-Jiang) [reply spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md:4]: fixed

**Review by ederks85 — APPROVED** [review]
LGTM

**Review by radcortez — APPROVED** [review]

**Comment by Emily-Jiang** [comment]
> @Emily-Jiang can you please squash your commits before we merge this?

Squshed and merged.

**Pull request merged**